contrib/kind.yaml.j2

```diff
@@ -15,6 +15,7 @@ kubeadmConfigPatches:
     name: config
   apiServer:
     extraArgs:
+      "etcd-servers": "https://ovn-control-plane:2379"
```

Comment:
this is not stopping etcd to be installed, KIND uses kubeadm under the hood.
Installing an external etcd cluster is tricky https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/high-availability/#external-etcd-nodes

Reply:
@aojea my idea was not to install, configure, and use an external etcd server. Just point ovn-control-plane2's and ovn-control-plane3's API server to the local etcd server running on ovn-control-plane itself (setup by kind). That is, the etcd servers are still running on ovn-control-plane2 and ovn-control-plane3, but they are not used. We just use the one running on ovn-control-plane node.